ilc/client/Client.js

```diff
@@ -268,6 +267,7 @@ export class Client {
             mountRootParcel: singleSpa.mountRootParcel.bind(singleSpa),
             importParcelFromApp: parcelApi.importParcelFromApp.bind(this),
             getAllSharedLibNames: () => Promise.resolve(Object.keys(this.#configRoot.getConfig().sharedLibs)),
+            addErrorTransformer: this.#errorHandlerManager.addErrorTransformer.bind(this.errorHandlerManager),
```

Comment:
As i understand this method is required for plugin. Do we have ability not to expose it globally ?